app/javascript/article-form/articleForm.jsx

```diff
@@ -264,10 +265,10 @@ export class ArticleForm extends Component {
     });
   };
 
-  toggleModal = () => {
+  toggleModal = (showModal) => {
```

Comment:
refactor (blocking): I know we're using `showModal` in a few places, but `showModal(true)` makes more sense than `toggleModal(true)`, or `openModal`.

```suggestion
  openModal = (showModal) => {
  // or showModal = (isOpen) => {
```